src/client.js

```diff
@@ -17,15 +17,20 @@ const store = configureStore(initialState, baseHistory)
 const history = syncHistoryWithStore(baseHistory, store)
 const root = document.getElementById('app')
 
-const renderApp = () => (
+const renderApp = (renderProps) => (
   <AppContainer>
     <Provider store={store}>
-      <Router history={history} routes={routes} />
+      <Router {...renderProps} />
     </Provider>
   </AppContainer>
 )
```

Comment:
@kybarg Thanks 😄 I didn't know you could add a `key` property to the `<Router />` component. I was scratching my head trying to think how to solve this warning about changing routes when HMR is called.